### ISSUE TITLE
Onfido-Release have old pod dependencies 

### ISSUE THREAD

**Issue by IuriiIaremenko** [onfido/onfido-ios-sdk#21]
Hello!

We want to use you framework, however ```Onfido-Release``` pod have dependencies to old versions of Alamofire and ObjectMapper.
Downgrade of our pods is not an option. Could you please update Onfido-Release?

Pods error log:
```
[!] Unable to satisfy the following requirements:

- `Alamofire (~> 4.5)` required by `Podfile`
- `Alamofire (= 4.6.0)` required by `Podfile.lock`
- `Alamofire (< 4.5)` required by `Onfido-Release (2.1.3)`
- `ObjectMapper` required by `Podfile`
- `ObjectMapper (= 3.1.0)` required by `Podfile.lock`
- `ObjectMapper (~> 2.2)` required by `Onfido-Release (1.2.7)` 
```

**Comment by rfreitas** [comment]
Hi Lurii,

We are looking to get rid of `ObjectMapper` so that would solve one problem.
However, for Alamofire we have the optimistic operator ,`~>`, therefore `4.6.0` should work with the sdk.
Am I looking at something wrong?

**Comment by IuriiIaremenko** [comment]
Hi Ricardo,

Thanks for support. It would be nice to have as less as possible dependencies of your library: ObjectMapper could be replaced with native code in Swift 4, ```Alamofire < 4.5``` didn't work for us unfortunately, so we made some workarounds. 

For now we`re using Onfido with workaround, but we looking forward for updates. Thank you!

**Comment by rfreitas** [comment]
Thanks for letting us know you found a workaround! We will be looking into this soon.

**Comment by anuragajwani** [comment]
@IuriiIaremenko Since SDK `7.1.0` Alamofire has been removed. We recently released `7.2.0`. We suggest you update to the latest SDK version. Please let us know if continue to encounter issues using our latest SDK.

Thanks,
Anurag
